android/res/layout/view_less_results_notification.xml

```diff
@@ -18,17 +18,39 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 -->
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@+id/view_less_results_notification"
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:background="@drawable/view_less_results_background_yellow"
-    android:gravity="center_vertical|center_horizontal"
-    android:orientation="horizontal">
-    <ImageView
-        android:id="@+id/view_less_results_notification_image"
+    android:layout_height="wrap_content">
+
+    <LinearLayout
```

Comment:
@aldenml marcelinkaaa says that this nesting was the only way the layout would preserve the height that she wanted.